src/sys/windows/selector.rs

```diff
@@ -122,7 +127,12 @@ impl SockState {
                  * events that the user is interested in. Therefore, cancel the pending
                  * poll operation; when we receive it's completion package, a new poll
                  * operation will be submitted with the correct event mask. */
-                self.cancel()?;
+                let result = self.cancel();
```

Comment:
Let's just handle the result directly with:

```
if let Err(e) = self.cancel() {
    self.error = e.raw_os_error();
    return Err(e);
}
```

If errors are able to be ignored (after we stumble across *some* documentation 😖) we can just make it a `match`.

Reply:
ok